peachjam/admin.py

```diff
@@ -606,6 +669,21 @@ def get_fieldsets(self, request, obj=None):
 
         return fieldsets
 
+    def document_access_link(self, obj):
+        if obj and obj.id:
+            url = reverse(
+                f"admin:{obj._meta.app_label}_{obj._meta.model_name}_permissions",
+                args=[quote(obj.pk)],
+            )
+            return format_html(
+                '<a href="{}">{}</a>',
+                url,
+                _("Select restricted document access groups"),
+            )
+        return "-"
+
+    document_access_link.short_description = "Restricted document access"
```

Comment:
Translateable

Reply:
we haven't translated these for the other descriptions. I can do it for all of them

Reply:
on second thought, I can see that a lot of admin text hasn't been translated. I can open a separate PR for that